tests/api/v1/subjects/get.js

```diff
@@ -395,10 +395,25 @@ describe(`api: GET ${path}`, () => {
     });
   });
 
+  it('returns expected fields when requesting tags field', (done) => {
+    api.get(`${path}?fields=isPublished,name,tags`)
+    .set('Authorization', token)
+    .expect(constants.httpStatus.OK)
+    .end((err, res) => {
+      if (err) {
+        done(err);
+      }
+
+      expect(res.body[ZERO]).to.have.property('tags');
+      expect(res.body[ZERO]).to.have.property('name');
+      expect(res.body[ZERO]).to.have.property('isPublished');
```

Comment:
can the test check whether the tags have the expected values? ie. setup subject with tags: ['aaa', 'bbb'] and expect res.body.tags to equal ['aaa', 'bbb']